api/api.js

```diff
@@ -0,0 +1,35 @@
+/**
```

Comment:
This file is something ive written in the past. Basically it grabs all the files in the current folder except itself and calls
```
let router = express.Router();      //Create a new Router object
let newEndpoint = require(path.join(__dirname, file));
newEndpoint.init(router);           //Initialize the endpoints
app.use('/api/' + path.basename(file, '.js'), router); //Mount sub-api

so each file is a separate api endpoint, `/api/{filename}/{fn}`